zuul.d/end-to-end.yaml

```diff
@@ -51,7 +51,7 @@
       - ^.github
     run: ci/playbooks/e2e-run.yml
     vars:
-      cifmw_run_tests: false
+      cifmw_run_tests: true
```

Comment:
we can drop this line from here in favor of https://github.com/openstack-k8s-operators/ci-framework/blob/537c3af2fb90ec3b50d24ea5ae24fd84f9466ed6/scenarios/centos-9/ci.yml#L32